### ISSUE TITLE
crc doesn't start (Is 'crc daemon' running? Cannot reach daemon API: Get "http://unix/api/version": open \\.\pipe\crc-http: Access is denied.)

### ISSUE THREAD

**Issue by arbuhoo22** [code-ready/crc#3186]
### General information

  * OS: Windows 11
  * Hypervisor: Hyper-V
  * Did you run `crc setup` before starting it (Yes/No)? Yes
  * Running CRC on: Laptop

## CRC version
```bash
C:\Users\Administrator>crc version
CRC version: 2.3.0+502bf1d9
OpenShift version: 4.10.12
Podman version: 3.4.4
```
  
## CRC status
```bash
C:\Users\Administrator>crc status --log-level debug
DEBU CRC version: 2.3.0+502bf1d9
DEBU OpenShift version: 4.10.12
DEBU Podman version: 3.4.4
DEBU Running 'crc status'
DEBU Checking file: C:\Users\Administrator\.crc\machines\crc\.crc-exist
DEBU Running '(Get-ItemProperty -Path "HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion" -Name ReleaseId).ReleaseId'
DEBU Running '(Get-ItemProperty -Path "HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion").EditionID'
Machine does not exist. Use 'crc start' to create it
```

## CRC config
```bash
C:\Users\Administrator>crc config view
- consent-telemetry                     : yes
```

## Host Operating System
```bash
OS Name	Microsoft Windows 11 Pro
Version	10.0.22000 Build 22000
Other OS Description 	Not Available
OS Manufacturer	Microsoft Corporation
System Name	XXXXXXXXXXXX
System Manufacturer	Dell Inc.
System Model	Latitude 5420
System Type	x64-based PC
System SKU	0A20
Processor	11th Gen Intel(R) Core(TM) i7-1165G7 @ 2.80GHz, 1690 Mhz, 4 Core(s), 8 Logical Processor(s)
BIOS Version/Date	Dell Inc. 1.17.2, 3/28/2022
SMBIOS Version	3.2
Embedded Controller Version	255.255
BIOS Mode	UEFI
BaseBoard Manufacturer	Dell Inc.
BaseBoard Product	035X97
BaseBoard Version	A00
Platform Role	Mobile
Secure Boot State	On
PCR7 Configuration	Elevation Required to View
Windows Directory	C:\WINDOWS
System Directory	C:\WINDOWS\system32
Boot Device	\Device\HarddiskVolume1
Locale	United States
Hardware Abstraction Layer	Version = "10.0.22000.527"
User Name	DESKTOP-75RHKDN\Administrator
Time Zone	SA Pacific Standard Time
Installed Physical Memory (RAM)	32.0 GB
Total Physical Memory	31.4 GB
Available Physical Memory	19.9 GB
Total Virtual Memory	40.4 GB
Available Virtual Memory	27.6 GB
Page File Space	9.00 GB
Page File	C:\pagefile.sys
Kernel DMA Protection	On
Virtualization-based security	Running
Virtualization-based security Required Security Properties	
Virtualization-based security Available Security Properties	Base Virtualization Support, Secure Boot, DMA Protection, UEFI Code Readonly, SMM Security Mitigations 1.0, Mode Based Execution Control, APIC Virtualization
Virtualization-based security Services Configured	
Virtualization-based security Services Running	
Device Encryption Support	Elevation Required to View
A hypervisor has been detected. Features required for Hyper-V will not be displayed.	

```

### Steps to reproduce

  1. Install CRC (crc-windows-amd64.msi).
  2.  Restart the machine
  3. Open "Red Hat Openshift Local"
      
![image](https://user-images.githubusercontent.com/37216731/169561934-0e215662-aa31-4d18-ac4d-054a0e1782b9.png)

  5. Follow the config wizzard.
  6. Configuration finish successfully.
  7. Start CRC, nothing happend.
      
![image](https://user-images.githubusercontent.com/37216731/169562872-617de645-badf-4b50-9e33-fab2c599a04c.png)

  8. Start CRC from command:
  
![image](https://user-images.githubusercontent.com/37216731/169563140-1e4e8f5e-686b-4c3e-b21a-c55932afbe0b.png)

### Expected
CRC starts. 

### Actual
C:\Users\Administrator>crc start --log-level debug
DEBU CRC version: 2.3.0+502bf1d9
DEBU OpenShift version: 4.10.12
DEBU Podman version: 3.4.4
DEBU Running 'crc start'
DEBU Total memory of system is 33723650048 bytes
DEBU No new version available. The latest version is 2.3.0
DEBU Checking file: C:\Users\Administrator\.crc\machines\crc\.crc-exist
DEBU Running '(Get-ItemProperty -Path "HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion" -Name ReleaseId).ReleaseId'
DEBU Running '(Get-ItemProperty -Path "HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion").EditionID'
Is 'crc daemon' running? Cannot reach daemon API: Get "http://unix/api/version": open \\.\pipe\crc-http: Access is denied.

### Logs

I ran:
$ crc delete -f
$ crc cleanup
$ crc setup
$ crc start --log-level debug

Logs files here: 
[crc.log](https://github.com/code-ready/crc/files/8741661/crc.log)
[crcd.log](https://github.com/code-ready/crc/files/8741655/crcd.log)

**Comment by praveenkumar** [comment]
@arbuhoo22 I see two different things as part of `crc setup` in the log file which you shared.

First `crc setup` does successful and daemon was running
```
time="2022-05-20T10:03:15-05:00" level=info msg="Checking if daemon task is running"
time="2022-05-20T10:03:15-05:00" level=debug msg="Running '(Get-ScheduledTask -TaskName \"crcDaemon\").State'"
time="2022-05-20T10:03:16-05:00" level=debug msg="expected crcDaemon task to be in 'Running' but got 'Ready\r\n'"
time="2022-05-20T10:03:16-05:00" level=info msg="Running the daemon task"
time="2022-05-20T10:03:16-05:00" level=debug msg="Running 'Start-ScheduledTask -TaskName crcDaemon'"
time="2022-05-20T10:03:16-05:00" level=debug msg="retry loop: attempt 0"
time="2022-05-20T10:03:16-05:00" level=debug msg="error: Temporary error: daemon is not running yet - sleeping 2s"
time="2022-05-20T10:03:18-05:00" level=debug msg="retry loop: attempt 1"
time="2022-05-20T10:03:18-05:00" level=info msg="Checking admin helper service is running"
time="2022-05-20T10:03:18-05:00" level=debug msg="Running '(Get-Service crcAdminHelper).Status'"
time="2022-05-20T10:03:19-05:00" level=debug msg="Running '(Get-ItemProperty -Path \"HKLM:\\SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\" -Name ReleaseId).ReleaseId'"
time="2022-05-20T10:03:19-05:00" level=debug msg="Running '(Get-ItemProperty -Path \"HKLM:\\SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\").EditionID'"
time="2022-05-20T10:04:26-05:00" level=debug msg="CRC version: 2.3.0+502bf1d9\n"
time="2022-05-20T10:04:26-05:00" level=debug msg="OpenShift version: 4.10.12\n"
```

Second time may be after delete of cleanup it have some issue.
```
ime="2022-05-20T10:42:35-05:00" level=debug msg="Running '(Get-ScheduledTask -TaskName \"crcDaemon\").Version'"
time="2022-05-20T10:42:36-05:00" level=info msg="Checking if daemon task is running"
time="2022-05-20T10:42:36-05:00" level=debug msg="Running '(Get-ScheduledTask -TaskName \"crcDaemon\").State'"
time="2022-05-20T10:42:37-05:00" level=debug msg="expected crcDaemon task to be in 'Running' but got 'Ready\r\n'"
time="2022-05-20T10:42:37-05:00" level=info msg="Running the daemon task"
time="2022-05-20T10:42:37-05:00" level=debug msg="Running 'Start-ScheduledTask -TaskName crcDaemon'"
time="2022-05-20T10:42:37-05:00" level=debug msg="retry loop: attempt 0"
time="2022-05-20T10:42:37-05:00" level=debug msg="error: Temporary error: daemon is not running yet - sleeping 2s"
time="2022-05-20T10:42:39-05:00" level=debug msg="retry loop: attempt 1"
time="2022-05-20T10:42:39-05:00" level=debug msg="error: Temporary error: daemon is not running yet - sleeping 2s"
time="2022-05-20T10:42:41-05:00" level=debug msg="retry loop: attempt 2"
time="2022-05-20T10:42:41-05:00" level=debug msg="error: Temporary error: daemon is not running yet - sleeping 2s"
time="2022-05-20T10:42:43-05:00" level=debug msg="retry loop: attempt 3"
time="2022-05-20T10:42:43-05:00" level=debug msg="error: Temporary error: daemon is not running yet - sleeping 2s"
time="2022-05-20T10:42:45-05:00" level=debug msg="retry loop: attempt 4"
time="2022-05-20T10:42:45-05:00" level=debug msg="error: Temporary error: daemon is not running yet - sleeping 2s"
time="2022-05-20T10:42:47-05:00" level=debug msg="retry loop: attempt 5"
time="2022-05-20T10:42:47-05:00" level=debug msg="error: Temporary error: daemon is not running yet - sleeping 2s"
time="2022-05-20T10:42:49-05:00" level=debug msg="retry loop: attempt 6"
time="2022-05-20T10:42:49-05:00" level=debug msg="error: Temporary error: daemon is not running yet - sleeping 2s"
time="2022-05-20T10:42:51-05:00" level=debug msg="retry loop: attempt 7"
time="2022-05-20T10:42:51-05:00" level=debug msg="error: Temporary error: daemon is not running yet - sleeping 2s"
time="2022-05-20T10:42:53-05:00" level=debug msg="RetryAfter timeout after 8 tries"
```

Can you try following in the powershell and let us know output?
```
> crc.exe cleanup --log-level debug
> crc.exe setup --log-level debug
```
Check the `crc` process from the task manager and see what all are running?

**Comment by arbuhoo22** [comment]
Hi @praveenkumar,

Thanks a lot for your answer. Here the logs after I executed the comands.
[crc.log](https://github.com/code-ready/crc/files/8755403/crc.log)

Task Manager
![image](https://user-images.githubusercontent.com/37216731/169842489-997dece2-3088-4a49-8ae1-ffca7fe076eb.png)




**PS C:\Users\Administrator> crc.exe cleanup --log-level debug**
DEBU CRC version: 2.3.0+502bf1d9
DEBU OpenShift version: 4.10.12
DEBU Podman version: 3.4.4
DEBU Running 'crc cleanup'
INFO Removing daemon task
DEBU Running 'Get-ScheduledTask -TaskName crcDaemon'
DEBU Running '(Get-ScheduledTask -TaskName "crcDaemon").State'
DEBU Running 'Stop-ScheduledTask -TaskName crcDaemon'
DEBU Running 'Get-ScheduledTask -TaskName crcDaemon'
DEBU Running '(Get-ScheduledTask -TaskName "crcDaemon").Version'
DEBU Running 'Unregister-ScheduledTask -TaskName crcDaemon -Confirm:$false'
DEBU Command failed: exit status 1
DEBU stdout:
DEBU stderr: Unregister-ScheduledTask : Access is denied.
At line:1 char:43
+ ... yContinue'; Unregister-ScheduledTask -TaskName crcDaemon -Confirm:$fa ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : PermissionDenied: (MSFT_ScheduledT...TaskPath = "\"):Root/Microsoft/...T_ScheduledTask)
   [Unregister-ScheduledTask], CimException
    + FullyQualifiedErrorId : HRESULT 0x80070005,Unregister-ScheduledTask

DEBU unable to unregister the crcDaemon task: exit status 1 : Unregister-ScheduledTask : Access is denied.
At line:1 char:43
+ ... yContinue'; Unregister-ScheduledTask -TaskName crcDaemon -Confirm:$fa ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : PermissionDenied: (MSFT_ScheduledT...TaskPath = "\"):Root/Microsoft/...T_ScheduledTask)
   [Unregister-ScheduledTask], CimException
    + FullyQualifiedErrorId : HRESULT 0x80070005,Unregister-ScheduledTask

DEBU exit status 1
INFO Removing hosts file records added by CRC
INFO Removing pull secret from the keyring
INFO Removing older logs
INFO Removing CRC Machine Instance directory
DEBU Deleting machines directory
INFO Removing crc's virtual machine
DEBU Running 'Get-VM -Name crc'
DEBU Command failed: exit status 1
DEBU stdout:
DEBU stderr: Get-VM : Hyper-V was unable to find a virtual machine with name "crc".
At line:1 char:43
+ $ProgressPreference = 'SilentlyContinue'; Get-VM -Name crc
+                                           ~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (crc:String) [Get-VM], VirtualizationException
    + FullyQualifiedErrorId : InvalidParameter,Microsoft.HyperV.PowerShell.Commands.GetVM

DEBU Running '(Get-ItemProperty -Path "HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion" -Name ReleaseId).ReleaseId'
DEBU Running '(Get-ItemProperty -Path "HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion").EditionID'
exit status 1


**PS C:\Users\Administrator> crc.exe setup --log-level debug**
DEBU CRC version: 2.3.0+502bf1d9
DEBU OpenShift version: 4.10.12
DEBU Podman version: 3.4.4
DEBU Running 'crc setup'
INFO Using bundle path C:\Users\Administrator\.crc\cache\crc_hyperv_4.10.12_amd64.crcbundle
INFO Checking if current user is in Hyper-V Admins group
DEBU Running '$sid = New-Object System.Security.Principal.SecurityIdentifier("S-1-5-32-578")
        @([Security.Principal.WindowsPrincipal][Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole($sid)'
INFO Checking if CRC bundle is extracted in '$HOME/.crc'
INFO Checking if C:\Users\Administrator\.crc\cache\crc_hyperv_4.10.12_amd64.crcbundle exists
DEBU C:\Users\Administrator\.crc\cache\crc_hyperv_4.10.12_amd64.crcbundle exists
INFO Checking if daemon task is installed
DEBU Running 'Get-ScheduledTask -TaskName crcDaemon'
DEBU Running '(Get-ScheduledTask -TaskName "crcDaemon").Version'
INFO Checking if daemon task is running
DEBU Running '(Get-ScheduledTask -TaskName "crcDaemon").State'
DEBU expected crcDaemon task to be in 'Running' but got 'Ready
'
INFO Running the daemon task
DEBU Running 'Start-ScheduledTask -TaskName crcDaemon'
DEBU retry loop: attempt 0
DEBU error: Temporary error: daemon is not running yet - sleeping 2s
DEBU retry loop: attempt 1
DEBU error: Temporary error: daemon is not running yet - sleeping 2s
DEBU retry loop: attempt 2
DEBU error: Temporary error: daemon is not running yet - sleeping 2s
DEBU retry loop: attempt 3
DEBU error: Temporary error: daemon is not running yet - sleeping 2s
DEBU retry loop: attempt 4
DEBU error: Temporary error: daemon is not running yet - sleeping 2s
DEBU retry loop: attempt 5
DEBU error: Temporary error: daemon is not running yet - sleeping 2s
DEBU retry loop: attempt 6
DEBU error: Temporary error: daemon is not running yet - sleeping 2s
DEBU retry loop: attempt 7
DEBU error: Temporary error: daemon is not running yet - sleeping 2s
DEBU RetryAfter timeout after 8 tries
DEBU Running '(Get-ItemProperty -Path "HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion" -Name ReleaseId).ReleaseId'
DEBU Running '(Get-ItemProperty -Path "HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion").EditionID'
Temporary error: daemon is not running yet (x8)

**Comment by praveenkumar** [comment]
@arbuhoo22 looks like you created the task using system administrator so better to remove using `Task Scheduler` app manually and try again with `crc cleanup` => `crc setup` 

```
DEBU Running 'Unregister-ScheduledTask -TaskName crcDaemon -Confirm:$false'
DEBU Command failed: exit status 1
DEBU stdout:
DEBU stderr: Unregister-ScheduledTask : Access is denied.
At line:1 char:43
+ ... yContinue'; Unregister-ScheduledTask -TaskName crcDaemon -Confirm:$fa ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : PermissionDenied: (MSFT_ScheduledT...TaskPath = "\"):Root/Microsoft/...T_ScheduledTask)
   [Unregister-ScheduledTask], CimException
    + FullyQualifiedErrorId : HRESULT 0x80070005,Unregister-ScheduledTask

DEBU unable to unregister the crcDaemon task: exit status 1 : Unregister-ScheduledTask : Access is denied.
At line:1 char:43
+ ... yContinue'; Unregister-ScheduledTask -TaskName crcDaemon -Confirm:$fa ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : PermissionDenied: (MSFT_ScheduledT...TaskPath = "\"):Root/Microsoft/...T_ScheduledTask)
   [Unregister-ScheduledTask], CimException
    + FullyQualifiedErrorId : HRESULT 0x80070005,Unregister-ScheduledTask
```

**Comment by arbuhoo22** [comment]
@praveenkumar, After remove the task from task scheduler "crc setup" fails. It just works if I execute powershell as administrator.

**PS C:\WINDOWS\system32> crc setup --log-level debug**
crc : level=debug msg="CRC version: 2.3.0+502bf1d9\n"
At line:1 char:1
+ crc setup --log-level debug
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (level=debug msg...3.0+502bf1d9\n":String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
 
level=debug msg="OpenShift version: 4.10.12\n"
level=debug msg="Podman version: 3.4.4\n"
level=debug msg="Running 'crc setup'"
level=info msg="Using bundle path C:\\Users\\Administrator\\.crc\\cache\\crc_hyperv_4.10.12_amd64.crcbundle"
level=info msg="Checking if current user is in Hyper-V Admins group"
level=debug msg="Running '$sid = New-Object System.Security.Principal.SecurityIdentifier(\"S-1-5-32-578\")\n\t@([Security.Principal.WindowsPrincipal][Security.Principal.
WindowsIdentity]::GetCurrent()).IsInRole($sid)'"
level=info msg="Checking if CRC bundle is extracted in '$HOME/.crc'"
level=info msg="Checking if C:\\Users\\Administrator\\.crc\\cache\\crc_hyperv_4.10.12_amd64.crcbundle exists"
level=debug msg="C:\\Users\\Administrator\\.crc\\cache\\crc_hyperv_4.10.12_amd64.crcbundle exists"
level=info msg="Checking if daemon task is installed"
level=debug msg="Running 'Get-ScheduledTask -TaskName crcDaemon'"
level=debug msg="Command failed: exit status 1"
level=debug msg="stdout: "
level=debug msg="stderr: Get-ScheduledTask : No MSFT_ScheduledTask objects found with property 'TaskName' equal to 'crcDaemon'.  Verify the \r\nvalue of the property 
and retry.\r\nAt line:1 char:43\r\n+ ... reference = 'SilentlyContinue'; Get-ScheduledTask -TaskName crcDaemon\r\n+                                     
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\r\n    + CategoryInfo          : ObjectNotFound: (crcDaemon:String) [Get-ScheduledTask], CimJobException\r\n    + 
FullyQualifiedErrorId : CmdletizationQuery_NotFound_TaskName,Get-ScheduledTask\r\n \r\n"
level=debug msg="crcDaemon task is not installed: exit status 1 : Get-ScheduledTask : No MSFT_ScheduledTask objects found with property 'TaskName' equal to 'crcDaemon'. 
 Verify the \r\nvalue of the property and retry.\r\nAt line:1 char:43\r\n+ ... reference = 'SilentlyContinue'; Get-ScheduledTask -TaskName crcDaemon\r\n+                
                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\r\n    + CategoryInfo          : ObjectNotFound: (crcDaemon:String) [Get-ScheduledTask], CimJobException\r\n  
  + FullyQualifiedErrorId : CmdletizationQuery_NotFound_TaskName,Get-ScheduledTask\r\n \r\n"
level=debug msg="exit status 1"
level=info msg="Installing the daemon task"
level=debug msg="Running 'Get-ScheduledTask -TaskName crcDaemon'"
level=debug msg="Command failed: exit status 1"
level=debug msg="stdout: "
level=debug msg="stderr: Get-ScheduledTask : No MSFT_ScheduledTask objects found with property 'TaskName' equal to 'crcDaemon'.  Verify the \r\nvalue of the property 
and retry.\r\nAt line:1 char:43\r\n+ ... reference = 'SilentlyContinue'; Get-ScheduledTask -TaskName crcDaemon\r\n+                                     
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\r\n    + CategoryInfo          : ObjectNotFound: (crcDaemon:String) [Get-ScheduledTask], CimJobException\r\n    + 
FullyQualifiedErrorId : CmdletizationQuery_NotFound_TaskName,Get-ScheduledTask\r\n \r\n"
level=debug msg="crcDaemon task is not installed: exit status 1 : Get-ScheduledTask : No MSFT_ScheduledTask objects found with property 'TaskName' equal to 'crcDaemon'. 
 Verify the \r\nvalue of the property and retry.\r\nAt line:1 char:43\r\n+ ... reference = 'SilentlyContinue'; Get-ScheduledTask -TaskName crcDaemon\r\n+                
                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\r\n    + CategoryInfo          : ObjectNotFound: (crcDaemon:String) [Get-ScheduledTask], CimJobException\r\n  
  + FullyQualifiedErrorId : CmdletizationQuery_NotFound_TaskName,Get-ScheduledTask\r\n \r\n"
level=debug msg="Running 'Register-ScheduledTask -Xml '<?xml version=\"1.0\" encoding=\"UTF-16\"?>\n<Task version=\"1.3\" 
xmlns=\"http://schemas.microsoft.com/windows/2004/02/mit/task\">\n  <RegistrationInfo>\n    <Description>Run crc daemon as a task</Description>\n    
<Version>2.3.0</Version>\n  </RegistrationInfo>\n  <Settings>\n    <DisallowStartIfOnBatteries>false</DisallowStartIfOnBatteries>\n    
<StopIfGoingOnBatteries>false</StopIfGoingOnBatteries>\n    <Hidden>true</Hidden>\n    <MultipleInstancesPolicy>IgnoreNew</MultipleInstancesPolicy>\n    
<IdleSettings>\n      <Duration>PT10M</Duration>\n      <WaitTimeout>PT1H</WaitTimeout>\n      <StopOnIdleEnd>true</StopOnIdleEnd>\n      
<RestartOnIdle>false</RestartOnIdle>\n    </IdleSettings>\n    <UseUnifiedSchedulingEngine>true</UseUnifiedSchedulingEngine>\n  </Settings>\n  <Triggers>\n    
<LogonTrigger>\n      <UserId>DESKTOP-75RHKDN\\Administrator</UserId>\n    </LogonTrigger>\n  </Triggers>\n  <Actions Context=\"Author\">\n    <Exec>\n      
<Command>powershell.exe</Command>\n      <Arguments>-WindowStyle Hidden -Command &amp; &#39;C:\\Program Files\\Red Hat OpenShift Local\\crc.exe&#39; 
daemon</Arguments>\n    </Exec>\n  </Actions>\n</Task>\n' -TaskName crcDaemon'"
level=debug msg="Command failed: exit status 1"
level=debug msg="stdout: "
level=debug msg="stderr: Register-ScheduledTask : Access is denied.\r\nAt line:1 char:43\r\n+ ... yContinue'; Register-ScheduledTask -Xml '<?xml version=\"1.0\" 
encoding ...\r\n+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\r\n    + CategoryInfo          : PermissionDenied: 
(PS_ScheduledTask:Root/Microsoft/...S_ScheduledTask) [Register-Schedul \r\n   edTask], CimException\r\n    + FullyQualifiedErrorId : HRESULT 
0x80070005,Register-ScheduledTask\r\n \r\n"
level=debug msg="Running '(Get-ItemProperty -Path \"HKLM:\\SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\" -Name ReleaseId).ReleaseId'"
level=debug msg="Running '(Get-ItemProperty -Path \"HKLM:\\SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\").EditionID'"
failed to register crcDaemon task, exit status 1: Register-ScheduledTask : Access is denied.
At line:1 char:43
+ ... yContinue'; Register-ScheduledTask -Xml '<?xml version="1.0" encoding ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : PermissionDenied: (PS_ScheduledTask:Root/Microsoft/...S_ScheduledTask) [Register-Schedul 
   edTask], CimException
    + FullyQualifiedErrorId : HRESULT 0x80070005,Register-ScheduledTask

**Comment by praveenkumar** [comment]
> After remove the task from task scheduler "crc setup" fails. It just works if I execute powershell as administrator.

@arbuhoo22 I think it is working with administrator shell because you started it in that shell. Try to do following in the administrator shell `crc delete` => `crc cleanup --log-level debug` and then close that administrator shell and open it with normal user and again try `crc setup --log-level debug` .

**Comment by geojaya** [comment]
I am having the same issue on Windows 11 professional, PodmAN 1.31 AND crc 2.25

**Comment by praveenkumar** [comment]
@geojaya have you followed all the steps which is listed in this issue and still didn't resolve? Also please create a new issue instead of commenting on a close issue (in case listed suggestion doesn't work for you).

**Comment by RAANGAJI** [comment]
Same issue. 
--
crc version
CRC version: 2.25.0+97705a
OpenShift version: 4.13.6
Podman version: 4.4.4

--

I made the above recommendations: @arbuhoo22 looks like you created the task using system administrator so better to remove using Task Scheduler app manually and try again with crc cleanup => crc setup

**Comment by RAANGAJI** [comment]
Edición	Windows 10 Enterprise
Versión	22H2
19045.3324

**Comment by RAANGAJI** [comment]
My issue was solved according posted solution (To fix the issue we ran Set-ExecutionPolicy -ExecutionPolicy RemoteSigned from powershell as administrator) in the issue https://github.com/crc-org/crc/issues/3812
